samtranslator/model/api/api_generator.py

```diff
@@ -564,6 +566,15 @@ def _construct_api_domain(  # noqa: too-many-branches
             logical_id = "RecordSetGroup" + logical_id_suffix
 
             record_set_group = route53_record_set_groups.get(logical_id)
+
+            if self.individual_recordset:
+                return (
```

Comment:
nit: because we return now 4 of them, making it a named Tuple will be more convinient for future maintenance

Reply:
I like the idea. Will update it.